### PR TITLE
add ogles_gpgpu v0.2.4

### PR DESCRIPTION
**Pull request by headupinclouds**

### CODE REVIEW

File: cmake/projects/ogles_gpgpu/hunter.cmake [hunk 1]
```diff
@@ -116,7 +116,7 @@ hunter_add_version(
     URL
     "https://github.com/hunter-packages/ogles_gpgpu/archive/v0.2.4.tar.gz"
     SHA1
-    5cf6d2804c849e16f0c463f83c361a85d2d9ed43
+    ce38c35533fbb8f8efc30eb74d300598ee2132aa
```
Comment (ruslo) [inline cmake/projects/ogles_gpgpu/hunter.cmake]: https://github.com/ruslo/hunter/issues/1032#issuecomment-329116284 related?

**Comment by headupinclouds** [comment]
Interesting.  Yes, I think it is related.  I was pretty sure I had confirmed the SHA1 last night.

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2422
* https://travis-ci.org/ingenue/hunter/builds/276932884?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2429
* https://travis-ci.org/ingenue/hunter/builds/277166131?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.105

**Pull request merged**